app/main.py

```diff
@@ -1,6 +1,23 @@
+import requests
+from pprint import pprint
+import os
+from dotenv import load_dotenv
+
+load_dotenv()
+
+
 def get_weather() -> None:
-    # write your code here
-    pass
+    api_key = os.getenv("API_KEY")
+
+    url = "http://api.weatherapi.com/v1/current.json"
+    params = {
+        "key": api_key,
+        "q": "Paris"
```

Comment:
Code Style issue: The city name 'Paris' is hardcoded here. It would be better to have it as a constant at the top of the file, so that it can be easily changed when needed.